changelog.dd

```diff
@@ -358,6 +362,56 @@ assert(a[0] == 42);
 -------
 )
 
+$(LI $(LNAME2 inner-class, `std.traits.isInnerClass` was added to identify
+    Java-style nested classes with an accessible `outer` pointer)
+    $(P Classes that are nested inside other classes (and not inside functions)
+    and that don't define any `outer` symbol have an `outer` field which allows
+    to get and even set the instance of the outer class they are nested in.
+    $(REF isInnerClass, std, traits) allows to identify them.
+    )
+-------
+class Outer
+{
+    class Inner1 { }
+    class Inner2
+    {
+        int outer; // redefines outer, so the Outer instance is not accessible
+    }
+    static class Inner3 {}  // static, thus not nested
+}
+static assert(isInnerClass!(Outer.Inner1));
+static assert(!isInnerClass!(Outer.Inner2));
+static assert(!isInnerClass!(Outer.Inner3));
+-------
+)
+
+$(LI $(LNAME2 emplace-inner-class, `std.conv.emplace` no longer allows to emplace
+    Java-style nested classes without specifying a suitable `outer` pointer)
+    $(P If a class or struct is nested, `emplace` cannot initialize its context
+    pointer, making every use of context variables a segmentation fault; if
```

Comment:
> works only for classes, and only when the context is another class.

Oh I didn't know that the other cases were broken too.
How about just talking about this case then? e.g.

"If a class or struct is nested "->" If a class is nested within a class"